doc/guidelines/e2e.md

```diff
@@ -0,0 +1,102 @@
+## How to write tests
+
+### Locator Guidelines
+
+Before developing E2E tests make sure you have read the [locator guidelines](https://github.com/ElektraInitiative/PermaplanT/doc/guidelines/frontend-locators.md).
+
+### General rules
+
+- Be consistent and minimalistic.
+  - Use the same vocabulary as playwright (click, visible, etc.)
+  - Avoid using multiple different verbs for the same actions, keep your vocabulary small and precise.
+    Use Playwrights vocabulary.
+    This means to prefix methods with the actions from playwright (e.g when calling `xyz.click()` from playwright inside that method, name the method `click_xyz()` not a mix of press, click, push etc.)
+  - Don't indent more than one time.
+  - Don't make a complicated call stack higher than two from a page object.
+- Every test should be independent from other tests (concurrency).
+- Name inputs or objects you create SUT (System under Test) so they are clearly marked as test artifacts.
+
+### A typical workflow
+
+The usual workflow consists of three steps:
```

Comment:
Shouldn't the workflow begin from our use cases and then writing a protocol?

Reply:
Yes, but the workflow here is only specific to writing E2E tests and not bothering about the processes that happened before that. Its just out of scope I think.